lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -223,3 +240,18 @@ class PlatformViewManager {
     return result;
   }
 }
+
+DomElement _defaultFactory(
+  int viewId, {
+  Object? params,
+}) {
+  params!;
+  params as _DefaultFactoryParams;
+  return domDocument.createElement(params.tagName);
+}
+
+typedef _DefaultFactoryParams = Map<Object?, Object?>;
+
+extension on _DefaultFactoryParams {
```

Comment:
What a wacky syntax we can get with extensions, I love it (in small doses) xD

Reply:
I actually removed this because I was able to do the same thing using the existing `readX` extensions:
https://github.com/flutter/engine/blob/e590b24f3962fda3ec9144dcee3f7565b195839a/lib/web_ui/lib/src/engine/util.dart#L560